src/sites/link/kurogaze.net.js

```diff
@@ -1,7 +1,13 @@
 _.register({
   rule: {
-    host: /^st\.kurogaze\.net$/,
-    query: /r=(.+)/,
+    host: [
+      /^st\.kurogaze\.net$/,
+      /^s\.yukisubs\.com$/,
+    ],
+    query: [
+      /r=(.+)/,
+      /link=([^&]+)/,
+    ],
```

Comment:
In this case,
```javascript
rule: [
  {
    host: /^st\.kurogaze\.net$/,
    query: /r=(.+)/,
  },
  {
    host: /^s\.yukisubs\.com$/,
    query: /link=(.+)/,
  },
],
```
would be better.

We don't need `r` for `s.yukisubs.com` or `link` for `st.kurogaze.net`.